R/distributions.R

```diff
@@ -290,16 +290,27 @@ get_discrete_dist <- function(params){
   # --- check args
   check_dist(params)
 
-  if(!(params$dist %in% c("lnorm", "gamma"))) stop(paste0("Distribution recipe has not
-been defined for specified distribution type (dist = ", params$dist, ")"))
+  if(!(params$dist %in% c("lnorm", "gamma", "norm"))) {
+    stop(paste0("Distribution recipe has not been defined
+                for specified distribution type (dist = `",
+                params$dist, "`)"))
+  }
 
   # --- get discrete dist
 
   if(params$dist == "lnorm"){
     x <- stats::dlnorm(
-      1:params$max,
+      x       = 1:params$max,
       meanlog = params$mean,
-      sdlog = params$sd
```

Comment:
can you clarify whether distribution lists with `$dist = "lnorm"` should have elements `mean` and `sd` or `meanlog` and `sdlog`? i think we have some inconsistency... based on how you've written it here, these lists should have elements called `mean` and `sd`, but in `test-distributions.R` lines 54-59, it looks like the expected format would have `meanlog` and `sdlog` as the names...

whichever names we decide on, we should have an assertion for the correct mean/sd names in `check_dist()` when `$dist = "lnorm"` before the other checks.